shopping-cart/shopping-cart-scala/project/plugins.sbt

```diff
@@ -1,5 +1,5 @@
 // The Lagom plugin
-addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.6.0-M6")
+addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.6.0-M7+51-a304aa13")
```

Comment:
This is a local snapshot build. I got hit by the `AkkaMaterializer` bug when running locally. 

Hopefully we get a 1.6.0-RC1 soon to move this forward